docs/application/dotnet/guides/machine-learning/pipeline.md

```diff
@@ -174,6 +174,127 @@ If you need to stop the data stream or select the data flow with multiple stream
     pipe.Start();
     ```
 
+## Custom Filter
+
+For your convenience, NNStreamer provides an interface for processing the tensor data with the `custom-easy` framework. After registering the user-defined callback method with the input and the output tensor information, NNStreamer can manipulate tensor data in the pipeline without an independent shared object. Since the callback method works as **filter** in the pipeline, it is named as `Custom Filter`.
```

Comment:
Should NNStreamer be in code tag?

Reply:
Hmmm... To emphasize a new concept, Custom Filter is highlighted. 
CAPI is also highlighted as below. If need to fix it, then let me know it.
* https://github.com/Samsung/tizen-docs/blob/tizen_6.0_prepare/docs/application/native/guides/machine-learning/machine-learning-inference.md#custom-filter

Reply:
@mijin85cho can you let us know if it is fine to leave the way as it is, what is your opinion?

Reply:
I think we can keep current state.